### DISCUSSION TITLE
visual studio 2019 report error:""Could Not Load“Microsoft.Bcl.AsyncInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51”或它的某一个依赖项。找到的程序集清单定义与程序集引用不匹配。 (异常来自 HRESULT:0x80131040)":"Microsoft.Bcl.AsyncInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"}System.IO.FileLoadException"

### DISCUSSION THREAD

**Discussion by citystrawman** [Tyrrrz/CliWrap#235] [Q&A] [answered]
Hi, I am using CliWrap in a visual studio 2019 project,  and I got an error saying` "{"未能加载文件或程序集“Microsoft.Bcl.AsyncInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51”或它的某一个依赖项。找到的程序集清单定义与程序集引用不匹配。 (异常来自 HRESULT:0x80131040)":"Microsoft.Bcl.AsyncInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"} | System.IO.FileLoadException"`
The characters means the system failed to load file/program. 

Here's my nuget installation info:
![image](https://github.com/Tyrrrz/CliWrap/assets/9004748/9ec74102-1b02-4280-a24b-7ec30722a880)

 I appreciate if anyone help me solve this issue, thank you!

**Comment by citystrawman** [comment, marked as answer]
Already solved this issue. downgrade AsyncInterfaces to v1.1.1 and then cliwrap will also automatically downgrade, then it works.(but the result do not have .isSuccess property)

**Comment by Tyrrrz** [comment]
It might be an issue related to your target framework. Are you building for the legacy .NET Framework?

**Reply by citystrawman** [reply]
I am using .NET Framework 4.7.2. Anyway I could run cliwrap now.
